scene/3d/skeleton_3d.cpp

```diff
@@ -103,6 +103,8 @@ bool Skeleton3D::_set(const StringName &p_path, const Variant &p_value) {
 		set_bone_pose_rotation(which, p_value);
 	} else if (what == "scale") {
 		set_bone_pose_scale(which, p_value);
+	} else if (what == "bonemeta") {
```

Comment:
```suggestion
	} else if (what == "bone_meta") {
```
Since the methods use `bone_meta` I'd say to use it everywhere

Reply:
Done